docs/rfc/examples/risingwave_ng.yaml

```diff
@@ -0,0 +1,233 @@
+apiVersion: risingwave.singularity-data.com/v1alpha1
```

Comment:
Do you think it's necessary to have a `complete.yml` file to demonstrate all variables defined?

Reply:
Though there're tons of fields in Pod's spec 🤣, yes, I think. Now, this file is also for the purpose of showing all variables.